index.html

```diff
@@ -241,7 +243,11 @@
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
     <script src='https://cdnjs.cloudflare.com/ajax/libs/particles.js/2.0.0/particles.min.js'></script>
     <script src="assets/js/particle.js"></script>
-
+    <script>
+        function topFunction() {
+  document.documentElement.scrollTop = 0; 
+}
```

Comment:
Please be mindful of the indentation and linting. Use prettier extension if required